bundles/org.eclipse.passage.lic.api/src/org/eclipse/passage/lic/api/internal/registry/ServiceId.java

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+ * Copyright (c) 2020 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.lic.api.internal.registry;
+
+/**
+ * Implementation is expected to represent <i>data class</i>: with
+ * {@code hashCode} and {@code equals} overloaded basing on enclosed data.
+ * 
+ * @since 0.7
+ */
+public interface ServiceId {
```

Comment:
`Identifier` ?

Reply:
FQN will be `...registry.Identifier` which is not the case, it should be identifier for _service_. 

`ServiceIdentifier`?

Reply:
`ServiceIdentifier` looks better fr me